packages/client/src/api/github-cloud-replay.api.ts

```diff
@@ -31,15 +32,7 @@ export const getGitHubCloudReplayBaseTestRun = async ({
       }
     )
     .catch((error) => {
-      if (isAxiosError(error)) {
-        const errorMessage = error.response?.data?.message;
-
-        if (errorMessage) {
-          throw new Error(errorMessage);
```

Comment:
@alexivanov I think this may have been why we weren't getting full error information (API call that failed etc.) in reports-diff-action logs